tests/unit/dojson/test_dojson_hepnames.py

```diff
@@ -55,16 +58,195 @@ def test_dates(marcxml_to_json, json_to_marc):
+            <subfield code="d">2020</subfield>
+            <subfield code="e">CERN-ALPHA</subfield>
+            <subfield code="0">00001</subfield>
+            <subfield code="s">2014</subfield>
+            <subfield code="z">current</subfield>
+        </datafield>
+        <datafield tag="693" ind1=" " ind2=" ">
+            <subfield code="d">2012</subfield>
+            <subfield code="e">CERN-ALPHA</subfield>
+            <subfield code="0">00001</subfield>
+            <subfield code="s">2010</subfield>
+        </datafield>
+        ''',
+        [
+            {
+                'curated_relation': True,
+                'current': True,
+                'end_year': 2020,
+                'name': 'CERN-ALPHA',
+                'record': in_context_get_record_ref('00001'),
+                'start_year': 2014,
+            },
+            {
+                'curated_relation': True,
+                'current': False,
+                'end_year': 2012,
+                'name': 'CERN-ALPHA',
+                'record': in_context_get_record_ref('00001'),
+                'start_year': 2010,
+            },
+        ],
+        [
+            {
+                '0': 1,
+                'd': 2020,
+                'e': 'CERN-ALPHA',
+                's': 2014,
+                'z': 'current',
+            },
+            {
+                '0': 1,
+                'd': 2012,
+                'e': 'CERN-ALPHA',
+                's': 2010,
+            },
+        ],
+    ),
+    (
+        'simultaneous_experiments',
+        '''
+        <datafield tag="693" ind1=" " ind2=" ">
+            <subfield code="d">2013</subfield>
+            <subfield code="e">FIRST-SIMULTANEOUS</subfield>
+            <subfield code="e">SECOND-SIMULTANEOUS</subfield>
+            <subfield code="0">00001</subfield>
+            <subfield code="0">00002</subfield>
+            <subfield code="s">2015</subfield>
+        </datafield>
+        ''',
+        [
+            {
+                'curated_relation': True,
+                'current': False,
+                'end_year': 2013,
+                'name': 'FIRST-SIMULTANEOUS',
+                'record': in_context_get_record_ref('00001'),
+                'start_year': 2015,
+            },
+            {
+                'curated_relation': True,
+                'current': False,
+                'end_year': 2013,
+                'name': 'SECOND-SIMULTANEOUS',
+                'record': in_context_get_record_ref('00002'),
+                'start_year': 2015
+            },
+        ],
+        [
+            {
+                '0': 1,
+                'd': 2013,
+                'e': 'FIRST-SIMULTANEOUS',
+                's': 2015,
+            },
+            {
+                '0': 2,
+                'd': 2013,
+                'e': 'SECOND-SIMULTANEOUS',
+                's': 2015,
+            },
+        ],
+    ),
+
+]
+
+
+@pytest.mark.parametrize("data", EXPERIMENTS_DATA, ids=idfn)
```

Comment:
found an ugly way, using a lambda to run the get_ref function inside the fixture context...